src/python/pants/option/arg_splitter.py

```diff
@@ -282,6 +263,7 @@ def _at_flag(self) -> bool:
             bool(self._unconsumed_args)
             and self._unconsumed_args[-1].startswith("-")
             and not self._at_double_dash()
+            and not self._at_scope()
```

Comment:
Some built in goals have aliases that is a flag, so we need to treat those aliases as goals, not flags.